config/cortexa15/make_defs.mk

```diff
@@ -57,7 +57,7 @@ endif
 ifeq ($(DEBUG_TYPE),noopt)
 COPTFLAGS      := -O0
 else
-COPTFLAGS      := -O2
+COPTFLAGS      := -O2 -mcpu=cortrex-a15
```

Comment:
This is a bit off-topic. Any ARM-related stuff should be moved to its own PR.

Reply:
+1